### DISCUSSION TITLE
Anyone have success getting lat / long without the geocoding api?

### DISCUSSION THREAD

**Discussion by EastSideCode** [wellyshen/use-places-autocomplete#835] [Q&A]
When not using this package I can easily grab the lat/long as follows:

 let autocomplete = new google.maps.places.Autocomplete(inputElemHere);
 autocomplete.addListener('place_changed', onPlaceChanged);

function onPlaceChanged() {
 var place = autocomplete.getPlace();
 var lat = place.geometry.location.lat();
var lng = place.geometry.location.lng();
}

This doesn't contribute to the cost of paying for geocoding API requests. 

Is there no clear workaround in this package to achieve the same thing?

**Comment by EastSideCode** [comment]
Ahh never mind -- it was actually very easy here.

**Reply by waldothedeveloper** [reply]
Hey @EastSideCode , how did you figure it out? I was looking on google docs for the `autocomplete.getPlace()` to get a more structured address but I can't find how to do it with this library. Can you post what you did? [https://developers.google.com/maps/documentation/javascript/examples/places-autocomplete-addressform#:~:text=When%20the%20user%20selects%20an%20address%20from%20the%20pick%20list%2C%20your%20application%20can%20populate%20the%20address%20form.](url)

**Reply by waldothedeveloper** [reply]
So @EastSideCode , I was able to find out what I wanted :)
This is what I wanted: 
[https://github.com/wellyshen/use-places-autocomplete#:~:text=%2D%20an%20exception.-,getDetails,-Retrieves%20a%20great](url)

**Reply by codythatsme** [reply]
Hey @waldothedeveloper, that link seems to be dead. What was the solution here?

Cheers
